compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -136,8 +136,11 @@ static void addBufferizePasses(OpPassManager &passManager) {
 }
 
 static void addTileAndDistributePasses(
-    OpPassManager &pm, bool useFuseTensorPadWithConsumerPass = true) {
-  pm.addPass(createTileAndDistributeToWorkgroupsPass());
+    OpPassManager &pm, bool useFuseTensorPadWithConsumerPass = true,
+    linalg::DistributionMethod distributionMethod =
```

Comment:
Could we make the default `Cyclic` and make it explicit wherever it is not cyclic. `Cyclic` is always correct. Unless handled specifically by setting the number of workgroups correctly, the other options can lead to correctness issues.

Reply:
Dropped that part as discussed offline.